BaseWeapon/src/main/java/com/github/sef24sp4/baseweapon/BaseProjectilePlugin.java

```diff
@@ -0,0 +1,25 @@
+package com.github.sef24sp4.baseweapon;
+
+import com.github.sef24sp4.common.entities.IEntity;
+import com.github.sef24sp4.common.interfaces.IEntityManager;
+import com.github.sef24sp4.common.interfaces.IGameSettings;
+import com.github.sef24sp4.common.services.IGamePluginService;
+
+public class BaseProjectilePlugin implements IGamePluginService {
+	@Override
+	public void launch(IEntityManager entityManager, IGameSettings gameSettings) {
+
+	}
+
+	@Override
+	public void gameStart(IEntityManager entityManager, IGameSettings gameSettings) {
+
+	}
+
+	@Override
+	public void gameStop(IEntityManager entityManager, IGameSettings gameSettings) {
+		for (IEntity projectile : entityManager.getEntitiesByClass(BaseProjectile.class)) {
+			entityManager.removeEntity(projectile);
+		}
```

Comment:
Please make use of the `IEntityManager.removeEntitiesByClass()` once #77 has been merged.